common/remote/rmtsmtp.cpp

```diff
@@ -505,6 +506,20 @@ class CMailInfo
         validator.validateValue(subject.get(), "email subject");
     }
 
+    void convertToTLSSocket()
+    {
+        secureSocketContext.setown(createSecureSocketContext(ClientSocket));
```

Comment:
Not for this PR, but does this need the ability to provide a client certificate for mtls?  If so this will need enhanced configuration support (from a secret).